packages/block-library/src/blocks/slideshow/preview.js

```diff
@@ -146,3 +146,23 @@ const SlideshowPreview = class extends Component {
 };
 
 export default SlideshowPreview;
+
+const getPostSlideData = ( posts, props ) => {
+  return (
+    posts.map( ( post, id ) => {
+      return (
+        <Slide id={id} className="novablocks-slideshow__slide" {...props} image={ wp.data.select( 'core').getMedia(post.featured_media) } post={ post } type="post"/>
```

Comment:
the image prop should be standardized before passing it to the Slide component and not inside it

Reply:
Also... the best case scenario is when the Slide component receives only two props `media` and `content`. You could try to reach this scenario